packages/scripts/templates/package.json.tmpl

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "${featureName.dashCase}-feature",
+  "version": "1.0.0",
+  "main": "feature/index",
```

Comment:
no no..
main should go to src/index

Reply:
but we don't have src folder here..

Reply:
yes, and create empty index.ts file there